test/unit/spec/function_output/kbaseAttributeMapping-spec.js

```diff
@@ -1,30 +1,33 @@
-define(['jquery', 'kbaseAttributeMapping', 'base/js/namespace', 'narrativeMocks'], (
-    $,
-    kbaseAttributeMapping,
-    Jupyter,
-    Mocks
-) => {
+define([
```

Comment:
See also the `kbaseAttributeMapping.js` code.
Refactored these to work consistently with the other changes.

I also tested them against some sample data. They're KBaseExperiments.AttributeMapping objects.